documentation/airport-guides/cyhz.md

```diff
@@ -0,0 +1,150 @@
+### About Halifax
+Halifax Stanfield International Airport is a Canadian airport serving Halifax and mainland Nova Scotia.
+Opened in June 1960, it is now a hub for Air Canada and Air Canada Express.
+
+### Statistics
+3,908,799 passengers in 2015-2016
+80,278 aircraft movement in 2015-2016
+
+### Positions
+Not much is known about Halifax due to lack of local knowledge, however from [LiveATC feeds](https://www.liveatc.net/search/?icao=CYHZ), we know that Halifax Terminal on frequency 119.200MHz is used most.
+
+|Callsign             | Frequency  | Remarks
+|---------------------|:----------:|:---------------------------------------------:|
+|"HALIFAX DELIVERY"   | 123.950MHz |                                               |
+|"HALIFAX GROUND"     | 121.900MHz |                                               |
+|"HALIFAX TOWER"      | 118.400MHz |                                               |
+|                     |            |                                               |
+|"HALIFAX TERMINAL"   | 119.200MHz | Frequency used for both departure and arrival |
+|"HALIFAX TERMINAL"   | 118.700MHz | Looks to be seldom used                       |
+|"HALIFAX TERMINAL"   | 128.550MHz | Low altitude, looks to be rarely used         |
+|                     |            |                                               |
+|"HALIFAX ATIS"       | 118.025MHz |                                               |
+
+### About the Halifax Terminal Control Areas
+NAVCanada outlines the airspace boundary around Halifax in their Designated Airspace Handbook. All radar positions (departure, arrival) surrounding Halifax have the callsign "Halifax Terminal". Aircraft leaving the Halifax TCA should be handed over to Moncton Centre.
+
+[This diagram](http://prntscr.com/hv4bf4) may assist in visualising the airspace around Halifax.
+
+### Instrument Procedures
+Halifax has one Standard Instrument Departure and four Standard Terminal Arrival Routes.
+
+##### Departures
+All departing aircraft will be assigned the Halifax Four departure. The Halifax Four advises pilots to climb to 5000ft on runway heading. Upon receipt of an aircraft on departure, Halifax Terminal should: 
+* give a direct to a point along the filed route;
+* give vectors to a point along the filed route;
+* issue climb instructions.
+
+##### Arrivals
+Moncton Centre will assign STARs to aircraft and will give initial descent instructions.
+
+Halifax Terminal is responsible for the following:
+* issuing further descent clearances;
+* clearing aircraft onto a final approach, ILS, RNAV, Visual or LOC;
+* ensuring aircraft have sufficient spacing between them.
+
+Where possible, Terminal should issue descent clearances to meet the following restrictions:
+- Runway 05

```

Comment:
For these restrictions, using just the initial approach fixes for the ILS's might be better (i.e. `LEROS`, `TETAR`, `ODKAS` and `VOKIL`)

Reply:
@Fechulo From what I've heard from listening to LiveATC, I believe they issue descent instructions to these altitudes first, but not totally sure

Reply:
Right, but I'm guessing they prefer to vector aircraft onto the ILS rather than clearing them for RNAV transitions. So while ATC give descent clearances to these altitudes, they aren't set restrictions. They're probably intermediate altitudes to comply with MVA's and such

Reply:
@Fechulo I can go back to LiveATC and listen to some of their archives but I heard more transitions than vectors the last time I was on there

Reply:
Well you did more research than I did, so you know what's best.